dev/Scroller/APITests/ScrollerSnapPointTests.cs

```diff
@@ -75,6 +74,13 @@ public void OverlappingRegularSnapPointsThrow()
         [TestProperty("Description", "Add and remove snap points and make sure the corresponding collections look correct.")]
         public void CanAddAndRemoveSnapPointsFromAScroller()
         {
+            CanAddAndRemoveSnapPointsFromAScroller(ScrollSnapPointsAlignment.Near);
+            CanAddAndRemoveSnapPointsFromAScroller(ScrollSnapPointsAlignment.Center);
+            CanAddAndRemoveSnapPointsFromAScroller(ScrollSnapPointsAlignment.Far);
+        }
+
+        private void CanAddAndRemoveSnapPointsFromAScroller(ScrollSnapPointsAlignment alignment)
```

Comment:
>private void CanAddAndRemoveSnapPointsFromAScroller(ScrollSnapPointsAlignment alignment) [](start = 8, length = 88)

You can mix alignments, have 1 snap point near, one mid, one far. Might be worth adding a test that has all three going at once.

Reply:
Added a new test that adds a mix of scroll snap points.